core-bundle/contao/templates/forms/form_wrapper.html5

```diff
@@ -6,19 +6,15 @@
     <<?= $this->hl ?>><?= $this->headline ?></<?= $this->hl ?>>
   <?php endif; ?>
 
-  <form<?php if ($this->action): ?> action="<?= $this->action ?>"<?php endif; ?> method="<?= $this->method ?>" enctype="<?= $this->enctype ?>"<?= $this->attributes ?><?= $this->novalidate ?>>
-    <div class="formbody">
-      <?php if ('get' != $this->method): ?>
-        <input type="hidden" name="FORM_SUBMIT" value="<?= $this->formSubmit ?>">
-        <input type="hidden" name="REQUEST_TOKEN" value="<?= $this->requestToken ?>">
-        <?php if ($this->maxFileSize): ?>
-          <input type="hidden" name="MAX_FILE_SIZE" value="<?= $this->maxFileSize ?>">
-        <?php endif; ?>
-      <?php endif; ?>
-      <?= $this->hidden ?>
-      <?= $this->fields ?>
-    </div>
-  </form>
+  <?php if ($this->confirmation): ?>
+    <?php $this->insert('form_confirmation', $this->arrData) ?>
+  <?php else: ?>
+    <?php $this->insert('form_inline', $this->arrData) ?>
+  <?php endif; ?>
+
+  <?php if (!$this->action): ?>
+    <script src="bundles/contaocore/ajax-form.min.js"></script>
```

Comment:
This would need `$this->asset()` but yes, we don't have a `manifest.json` in the core-bundle yet.

Reply:
@contao/developers we need to decide how to proceed with this one. Imho, this is the only blocker in this PR because we want to have cache busting hashes here.
Currently, we do not yet have any option to use `$this->asset('....', 'contao_core')`. This is, because currently, basically all of our FE scripts are `contao-components` which are registered as asset package here: https://github.com/contao/contao/blob/5.x/core-bundle/src/DependencyInjection/Compiler/AddAssetsPackagesPass.php#L88
And I agree this makes sense for stuff like `tinymce` because there are probably different versions of that component and you want to decide on which one to use by requiring the component yourself in your `composer.json`.
But here, we don't have multiple versions. This feature is part of the core and I don't see why this would be a replacable component? 
Imho, we should introduce a build process for FE assets in the core too and make sure, we have a `manifest.json` in `public` as well. Only if we have this, we'll be able to provide assets too, see https://github.com/contao/contao/blob/5.x/core-bundle/src/DependencyInjection/Compiler/AddAssetsPackagesPass.php#L67.

For that, we'd need probably need to introduce Webpack/Symfony Encore alongside our Gulp process. Wdyt?

Reply:
I know, that's not the point though.

Reply:
Yes, just pointing out the possibilities for _this_ PR. I would love for Contao to support a build process for FE assets out of the box.

Reply:
I’d also use the suggestion from https://github.com/contao/contao/pull/5307#discussion_r980017468 for this PR and create a new PR for the FE build process assets topic.

Reply:
Done in df13bae6.